### PR TITLE
55841/feat/add-new-streets

### PR DESCRIPTION
**Pull request by chenFriedman**

### CODE REVIEW

**Review by ornahasson — COMMENTED** [review]
You should add on conflict to the queries

**Comment by chenFriedman** [comment]
> You should add on conflict to the queries

there is no need because its uniqe key

File: server/src/ScriptRunner/Scripts/index/add_new_streets.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+INSERT INTO public.streets (id, display_name, city, mho_code)
+VALUES ('100000', 'ירושלים', '681', '208');
```
Comment (barnahum8) [inline server/src/ScriptRunner/Scripts/index/add_new_streets.sql:2]: add on conflict

File: server/src/ScriptRunner/Scripts/index/add_new_streets.sql [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+INSERT INTO public.streets (id, display_name, city, mho_code)
+VALUES ('100001', 'קרן קיימת לישראל', '6500', '515');
```
Comment (barnahum8) [inline server/src/ScriptRunner/Scripts/index/add_new_streets.sql:5]: same

**Comment by barnahum8** [comment]
> 
> 
> > You should add on conflict to the queries
> 
> there is no need because its uniqe key

yes, but you still need to use on conflict instead of letting the query fail

**Review by ornahasson — APPROVED** [review]

**Pull request merged**